draft-irtf-cfrg-hpke.md

```diff
@@ -554,6 +554,42 @@ def Context.Open(aad, ct):
   return pt
 ~~~~~
 
+# Single-Shot APIs
+
+In many cases, applications encrypt only a single message to a recipient's public key. 
+This section provides templates for HPKE APIs that implement "single-shot" encryption 
+and decryption using APIs specified in {{hpke-kem}} and {{hpke-dem}}. Without loss
+of generality, assume the following function templates exist for single-shot encryption
```

Comment:
WLOG seems unnecessary.  I would just eliminate this sentence and end the prior sentence with a colon.

Reply:
Works for me!